english/marshmallow-taffy/start-conversation/conversation.md

```diff
@@ -1,4 +1,7 @@
 You greet Taffy politly, he responds with strange line:
-  Welcome back %username%
+  Welcome back friend!
 
 Do You:
+[Go exploring outside?](../explore-outside/explore-outside.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.